analysis/bin/main.ml

```diff
@@ -194,21 +194,23 @@ let main () =
     Sys.argv.(len - 1) <- "";
     Reanalyze.cli ()
   | [_; "references"; path; line; col] ->
-    Commands.references ~path
-      ~pos:(int_of_string line, int_of_string col)
-      ~debug
+    Eio_main.run (fun env ->
+        Commands.references ~env ~path
```

Comment:
why does references need to take an env now and not before?

Reply:
Because it's using Eio, and that requires the Eio env.